tests/Fixtures/PostFormation.php

```diff
@@ -139,6 +141,16 @@ public function slices(): array
         ];
     }
 
+    public function actions(): array
+    {
+        return [
+            Action::make('set-status')
+                ->job(SetStatus::class)
+                ->can('setStatus', Post::class)
```

Comment:
Can we add a test for this 

You can set a policy method to a config value like always be true like this:

return config('headless-testing-action-policy', true)

And in the test you set that to false 

config()->set(['headless-testing-action-policy' => false])

And assertForbidden

Also let's add a assert 404 on progress with a random id

Reply:
Done